chainer/functions/pooling/unpooling_2d.py

```diff
@@ -1,4 +1,9 @@
 import numpy
+import numpy.lib.stride_tricks
+try:
+    import cupy.lib.stride_tricks  # NOQA
+except ImportError:
```

Comment:
Could you change this line to catch the error with `Exception` ?
Because if the CuPy installation is failed, the broken CuPy may emit `AttributeError` when importing is failed. (ref: https://github.com/chainer/chainer/pull/6085)
```suggestion
except Exception:
```